src/shared-components/Recurrence/RecurrenceWeeklySection.tsx

```
@@ -0,0 +1,67 @@
+              if (e.key === 'Enter' || e.key === ' ') {
+                e.preventDefault();
+                onDayClick(Days[index]);
+              } else {
+                onWeekdayKeyDown(e, index);
+              }
+            }}
+            data-testid="recurrenceWeekDay"
+            data-cy={`recurrenceWeekDay-${index}`}
+            aria-pressed={byDay?.includes(Days[index])}
+            aria-label={`${t('select')} ${day}`}
+            role="button"
```

Biome (2.1.2)

[error] The elements with this role can be changed to the following elements:
<button>

For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)